lib/parse-message.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+const Client = require('./../model/client');
+const EE = require('events');
```

Comment:
Good job attempting how to do event emitters. I think I have an idea of what to do so you can use them properly in your server file so your chatroom can work as expected. Ask me if you need help on that.